ThirdParty/Bolts/BoltsInClasspath/BoltsInClasspath/main.m

```diff
@@ -0,0 +1,16 @@
+//
+//  main.m
+//  BoltsInClasspath
+//
+//  Created by siva kongara on 1/14/17.
+//  Copyright © 2017 venkat kongaraSiva. All rights reserved.
```

Comment:
Still missing the MIT license.

---
In reply to: [97418217](https://github.com/Microsoft/WinObjC-Samples/pull/12#discussion_r97418217) [](ancestors = 97418217)